build.gradle.kts

```diff
@@ -1,91 +1,93 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
-plugins {
-    kotlin("jvm") version "1.5.31"
-    kotlin("plugin.serialization") version "1.5.31"
-    id("org.jetbrains.kotlinx.kover") version "0.4.2"
-}
+buildscript {
+    val kotlinVersion = "1.5.31"
 
-group = "com.wire.kalium"
-version = "0.0.1-SNAPSHOT"
+    repositories {
+        google()
+        mavenCentral()
+    }
+
+    dependencies {
+        // keeping this here to allow AS to automatically update
+        classpath("com.android.tools.build:gradle:7.0.3")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlinVersion}")
+    }
+}
 
 repositories {
+    google()
     mavenCentral()
 }
 
 tasks.withType<Test> {
     useJUnitPlatform()
 }
 
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+    }
+}
+
 dependencies {
     val cryptoboxVersion = "1.0.0"
     val genericMessageProtoVersion = "1.35.0"
     val jacksonVersion = "2.12.3"
     val javaxValidationVersion = "2.0.1.Final"
     val kotestVersion = "4.6.3"
     val junitVersion = "5.7.1"
-    val kotlinVersion = "1.5.31"
     val mockkVersion = "1.12.1"
     val jakartaVersion = "2.1.6"
     val jerseyVersion = "2.32"
     val tyrusVersion = "1.13.1"
     val ktxSerializationVersion = "1.3.0"
-    val ktorVersion = "1.6.4"
     val coroutinesVersion = "1.5.2"
     val junit4Version = "4.13"
```

Comment:
Maybe move these lib versions to the `Dependencies` file?

Reply:
I left it on purpose for consultation.

But it kinda sucks.

I guess moving to `legacy` makes more sense.